alerting/pagerduty/README.md

```diff
@@ -61,3 +61,21 @@ variable "pagerduty_integration_key" {
 ## Notes
 
 * As for any integration configuration you need a **session** token from your SignalFx user (and not an **org** access token)
+
+## Usage
+
+```
+resource "signalfx_detector" "my_detector" {
+  // Detector stuff
+
+  rule {
+    description : "rule description"
+    severity      = "Severity"
+    detect_label  = "Detector Label ..."
+    notifications = [
+      format("PagerDuty,%s", module.signalfx-integrations-alerting-pagerduty.sfx_integration_id)
```

Comment:
I advice to use a local to make it possible to reuse it over lot of detectors instead of calling `format` as many times you have a detector to configure

Reply:
https://github.com/claranet/terraform-signalfx-integrations/blob/master/alerting/pagerduty/variables.tf#L18

Reply:
Add "locals" to readme.
I use "format" to don't use tf11 notation "${..}" anymore.